rest/src/main/java/cz/fi/muni/pa165/rest/controller/MushroomController.java

```diff
@@ -0,0 +1,71 @@
+package cz.fi.muni.pa165.rest.controller;
+
+import cz.fi.muni.pa165.dto.DateDTO;
+import cz.fi.muni.pa165.dto.DateIntervalDTO;
+import cz.fi.muni.pa165.dto.MushroomDTO;
+import cz.fi.muni.pa165.enums.MushroomType;
+import cz.fi.muni.pa165.facade.MushroomFacade;
+import cz.fi.muni.pa165.rest.Uris;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**
+ * @author Filip Krepinsky (410022) on 12/4/16
+ */
+@RestController
+@RequestMapping(path = Uris.ROOT_URI_MUSHROOM, produces = MediaType.APPLICATION_JSON_VALUE)
+public class MushroomController {
+
+    @Autowired
+    MushroomFacade mushroomFacade;
+
+    @RequestMapping(path = "/create", method = RequestMethod.POST)
+    public MushroomDTO create(@RequestBody MushroomDTO mushroomDTO) {
```

Comment:
Myslim ze ku vsetkym metodam by si mal pridat aj to aky mediaType konzumuju/produkuju